### PR TITLE
docs: Update README to contain more information about this repo

### PR DESCRIPTION
**Pull request by akhil888binoy**
made the readme file to look more legit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
-🔹Tailwind
+App Development: We develop custom mobile and web applications that cater to your specific business requirements and goals.
 
-🔹ContentLayer
+# Tech Stack
 
-Hosted on vercel 🔺
+The following tech stack was used to build this website:
+
+Next.js
```
Comment (devj3ns) [inline README.md:17]: I would suggest showing the techstack as an unordered list for better readability

File: README.md [hunk 2]
```diff
@@ -1,15 +1,37 @@
...
+TypeScript
+Tailwind
+ContentLayer
+
+# Contributing
+
+We welcome contributions from the community! If you would like to contribute to our projects, please follow these guidelines:
+
+Fork this repository
```
Comment (devj3ns) [inline README.md:26]: Same as above, but an ordered list here :)

**Review by devj3ns — CHANGES_REQUESTED** [review]
Hey @akhil888binoy, 
indeed, the README does not look very serious. With your changes, it's way better.

Thanks a lot👍

**Comment by akhil888binoy** [comment]
I have done the necessary changes...please check it out

**Comment by devj3ns** [comment]
Thanks, I have merged it :)

**Comment by akhil888binoy** [comment]
You are welcome

**Pull request merged**